public/v2/apps/wordpress-only.json

```diff
@@ -0,0 +1,58 @@
+{
+  "captainVersion": "2",
+  "documentation": "Taken from https://docs.docker.com/compose/wordpress/. Port mapping removed from WP as it is no longer needed",
+  "displayName": "WordPress",
+  "description": "This will create a WordPress only. You'll need to enter the database information and allow you to have a central DB for all your WP's.",
+  "dockerCompose": {
+    "version": "3.3",
+    "services": {
+      "$$cap_appname-wordpress": {
+        "image": "wordpress:$$cap_wp_version",
+        "volumes": ["$$cap_appname-wp-data:/var/www/html"],
+        "restart": "always",
+        "environment": {
+          "WORDPRESS_DB_HOST": "$$cap_db_host:3306",
+          "WORDPRESS_DB_NAME": "$$cap_db_name",
+          "WORDPRESS_DB_USER": "$$cap_db_user",
+          "WORDPRESS_DB_PASSWORD": "$$cap_db_pass"
+        }
+      }
+    }
+  },
+  "instructions": {
+    "start": "WordPress is an online, open source website creation tool written in PHP. But in non-geek speak, it’s probably the easiest and most powerful blogging and website content management system (or CMS) in existence today.\n\n Enter your WordPress Configuration parameters and click on next. A WordPress container will be created for you. The process will take about a minute for the process to finish. \n\nNote: This app is intended for advanced users who'd like to have a central DB in a single container for multiple WP's.",
```

Comment:
Add a bit of note to this re-highlighting the fact that it's without a database.

Reply:
Changed the description to make emphasis that this needs its own DB and that is intended for advanced users.